### PR TITLE
Rs 2022.11.19

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-11-19-155032_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 10/10_01_CIEL10_1_1.txt [hunk 1]
```diff
@@ -292,22 +315,23 @@ The thought of this worst-case scenario is heart-wrenching.
...
 -- Page 57, Offset 27219.
 -- それはあり得ない事だ、と否定するより、
-Is less the fact that it's impossible.
+I thought I'd be able to reject these hypotheticals.
 }
 [sha:fb56d4e66aa11f223da844454f96cbfaa5ac3c68]{
 -- Page 57, Offset 27220.
 -- それが当然なのだと肯定する自分がいる事だった。
-But rather that I agree it would be the natural thing to do.
+But instead, there's a part of me that thinks they're obvious truths.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_1_1.txt:334]: I see you've interpreted 自分がいる as "part of me" again. I don't think that's what it is.  It's just an indirect way of referring to oneself. Also obvious truths, and I dunno, this is a bit weird.

>But instead, I've already accepted that such an outcome was to be expected.

File: script/Ciel/Day 10/10_01_CIEL10_1_1.txt [hunk 2]
```diff
@@ -292,22 +315,23 @@ The thought of this worst-case scenario is heart-wrenching.
...
 }
 [sha:120aaadfc32150c59b70fd7efb442b71e66d3b32]{
 -- Page 58, Offset 27221.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_1_1.txt:337]: Hmmm, this sounds very strange.

>Stop thinking such stupid things...

File: script/Ciel/Day 10/10_01_CIEL10_1_1.txt [hunk 3]
```diff
@@ -317,32 +341,34 @@ But rather that I agree it would be the natural thing to do.
...
+ In any case, there's nothing for me to do but to wait.
 }
 [sha:50a728987a918f106c258cf1718886845647dcbe]{
 -- Page 59, Offset 27227.
 -- たとえこのまま、永遠に先輩が現れないとしても。
-Even if Senpai were to never appear―――
+I'll wait right here, even if she never comes...
 }
 [sha:6147a089919bd97ca4477cff9def504c7797c105]{
 -- Page 60, Offset 27228.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_1_1.txt:374]: see previous

**Review by Petrikow — COMMENTED** [review]
kjh

File: script/Ciel/Day 10/10_01_CIEL10_1.txt [hunk 4]
```diff
@@ -72,27 +75,31 @@ Thinking about what I am to her scares me.
...
 -- Page 4, Offset 27068.
 -- 俺は自分がどんな人間なのかを思い知った。
-I realized what kind of person I was.
+I realized what kind of person I am.
 }
 [sha:e270693e95d5f41e8756dbfa8a46c3c1afdc7142]{
 -- Page 4, Offset 27069.
 -- 酌量の余地はない。
-No room for extenuating circumstances.
+No allowance should be made for me.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_1.txt:90]: Hm, your suggestion reads like a different interpretation to the one I'd had. 
I read this as tied to the next line - he's a monster -> he shouldn't be allowed to exist here - but 'no room for debate' is more 'yup I'm a bad guy alright', which doesn't seem to capture the 酌量?

File: script/Ciel/Day 10/10_01_CIEL10_1.txt [hunk 5]
```diff
@@ -174,27 +186,31 @@ A horrible dull pain runs through my temple as soon as I open my eyes.
 [sha:9869f7b390943beb5558264c082e0d873306008e]{
 -- Page 11, Offset 27087.
 -- 「おはよう翡翠。今朝もありがとう。時間は……ちょっと余裕ないかな。すぐに着替えて食堂に行くよ」
-"Good morning, Hisui. Thanks for waking me today as well. Mmm, I still have some time left. I'll change right away and then go to the dining room."
+"Good morning, Hisui.
+ Thanks for waking me today as well.
+ What time is it... Oops, guess I'm cutting it close.
+ I'll change right away and then head to the dining room."
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 10/10_01_CIEL10_1.txt:192]: Eh, I feel like this is a more natural speech pattern in this context, and the gap in va gives some leeway.

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 10/10_01_CIEL10_1.txt:186]: It just sounded kinda lame, honestly. But it's not a big deal, if you feel like its better this way I don't particularly care.

**Pull request merged**